src/components/Sidebar/Sidebar.js

```diff
@@ -198,6 +216,7 @@ const Projects = styled.div.attrs({
   display: flex;
   flex-direction: column;
   align-items: center;
+  padding-top: 40px;
```

Comment:
I think we could modify this to add a top & bottom padding.
```suggestion
  padding: 40px 0;
```

To avoid a cutted plus icon at scroll end see screenshot below.
![grafik](https://user-images.githubusercontent.com/3046542/47549027-12498880-d8fb-11e8-87a1-2f59b7d055c6.png)